microsoft-365/commerce/licenses/add-licenses-using-product-key.md

```diff
@@ -23,7 +23,7 @@ ROBOTS: NOINDEX
 ms.date: 04/07/2021
 ---
 
-# Add licenses to or extend a subscription paid for using a product key
+# Add licenses to or extend a subscription paid for using a Microsoft 365 product key
 
```

Comment:
@lhcole613 - This is kind of ambiguous and confusing and I'm not sure what it's saying. I think the "subscription paid for" is throwing me off. Should this be "Add licenses to or extend a paid subscription by using a Microsoft 365 product key"? I think that's what it should be, based on the paragraph below the heading. But the way it's written, it sounds like it's saying you can add licenses to or extend a subscription that you used a product key to pay for?

Reply:
@lhcole613 Please use Sherry's suggested title: "Add licenses to or extend a paid subscription by using a Microsoft 365 product key"